Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -171,10 +171,10 @@ class BrowserLocationView: UIView {
     }
 
     private func updateTextWithURL() {
-        if let httplessURL = url?.absoluteStringWithoutHTTPScheme(), let baseDomain = url?.baseDomain() {
+        if let protocollessURL = url?.absoluteStringWithoutProtocol(), let baseDomain = url?.baseDomain() {
```

Comment:
As per bug https://bugzilla.mozilla.org/1189844, don't we have to change the URL bar to show only domain and subsequent folders?
As per my understanding TopSitesPanel, this is the code for showing all websites cells when user clicks on button which shows all websites.